src/smk/tool/geomark/tool-geomark.js

```diff
@@ -102,6 +120,13 @@ include.module( 'tool-geomark', [
                 }
             } )
 
+            this.createAndShowAlert = function(alertBodyArg) {
+                self.alertBody = alertBodyArg;
+                self.showAlert = true;
+            }
+
+            this.handlePrompt = function(promptValue) {};
```

Comment:
I use "handlePrompt" for dynamic handling. It's unnecessary now because there's only one prompt, but if we added another, we'd want the "close prompt" event to call whichever handler function we assigned before showing the prompt.

Do you think it's overkill to keep it as-is? This could then be reused elsewhere going forward. Or should we trust that we can implement "close prompt" handling in a reusable way if/when we need to going forward?

Reply:
We're highly unlikely to require an additional prompt in the Geomark component. I'd recommend simplifying in this case.

Reply:
Good idea to try and anticipate future needs! I do think it is highly unlikely we will add an additional prompt to the geomark component though. My personal preference would be for simplicity in this case.

Reply:
I'll remove the "handlePrompt" variable then.